poros/game_results.rb

```diff
@@ -0,0 +1,64 @@
+
+class GameResults
+  
+  def service 
+    GameService.new
+  end
+
+  def game_params_by_name(game_name)
+    params = service.get_games_by_name(game_name)[0]
+    binding.pry
+    result = {data: {
+      age_ratings:  run_array(:get_age_ratings, :rating, params[:age_ratings]), # array of ids
+      release_date: service.get_release_date(params[:first_release_date]), # id
+      cover:        cover_url(params[:cover]),
+      popularity:   params[:popularity], # good
+      summary:      params[:summary], # good
```

Comment:
some values are returned as usable data like summary as a string and popularity as a float. but others are just lists of ids [1,10, 43...]